### ISSUE TITLE
Default fees are incorrect

### ISSUE THREAD

**Issue by defunctec** [Crowndev/crown#246]
Fee's need a second look. Sending fails to add to mempool since the fee is lower than required.
To resolve, you must customise the fee within QT or settxfee to 0.001.

**Comment by BlockMechanic** [comment]
do you have ``` fallbackfee=0.001 ``` in your conf ?

**Comment by BlockMechanic** [comment]
I've bumped the fee up a bit, but this is not the ideal solution.

**Comment by defunctec** [comment]
> do you have `fallbackfee=0.001` in your conf ?

No. Is fallbackfee within default conf creation? If not, it should be.

**Comment by BlockMechanic** [comment]
It is now.
